cloud-byos/src/main/java/com/sequenceiq/cloudbreak/cloud/byos/BYOSConnector.java

```diff
@@ -0,0 +1,68 @@
+package com.sequenceiq.cloudbreak.cloud.byos;
+
+import javax.inject.Inject;
+
+import org.springframework.stereotype.Service;
+
+import com.sequenceiq.cloudbreak.cloud.Authenticator;
+import com.sequenceiq.cloudbreak.cloud.CloudConnector;
+import com.sequenceiq.cloudbreak.cloud.CredentialConnector;
+import com.sequenceiq.cloudbreak.cloud.InstanceConnector;
+import com.sequenceiq.cloudbreak.cloud.MetadataCollector;
+import com.sequenceiq.cloudbreak.cloud.PlatformParameters;
+import com.sequenceiq.cloudbreak.cloud.ResourceConnector;
+import com.sequenceiq.cloudbreak.cloud.Setup;
+import com.sequenceiq.cloudbreak.cloud.model.Platform;
+import com.sequenceiq.cloudbreak.cloud.model.Variant;
+
+@Service
+public class BYOSConnector implements CloudConnector {
```

Comment:
It is not a dummy implementation, but every method throws an exception, therefore I am not sure that we really need this.